src/collector.js

```diff
@@ -76,6 +77,18 @@ export default class Collector {
     return false;
   }
 
+  recordScannedFile(filename, scanner) {
+    // TODO: Add some code that verifies and normalizes `filename`
+    // to better avoid duplicates.
+    if (filename in this.scannedFiles) {
+      if (!this.scannedFiles[filename].includes(scanner)) {
+        this.scannedFiles[filename].push(scanner);
```

Comment:
I think this should be inverted; it's possible to run multiple scanners over the same file, as I understand it (and I think we do already).

I would make the key the `scannerName` and have it contain an array of `filename`s. That way you can see all the files scanned by each scanner. That makes more sense to me.